scholarx/2023/index.html

```diff
@@ -303,10 +297,8 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
               <div id="collapseThree" class="collapse" aria-labelledby="headingThree"
                    data-parent="#accordionExampleOne">
                 <div class="card-body opacity-8 text-left">
-                  Applications for the mentors of the ScholarX program 2023 will open on 13th of March 2023 and will close 14th of April
-                  2023. Keep an eye on the ScholarX page on our website and our social media handles for the application link.
-                  Join the "Mentor waiting list" with the button above to get notified when the applications are out",
-                  Otherwise, you may also submit your CV, a headshot, and expression of interest to <a
+                  Applications for the mentors of the ScholarX program 2023 will open on 13th of March 2023 and will close 11th of April 2023. 
+                  Since the application period is over, you may also submit your CV, a headshot, and expression of interest to <a
```

Comment:
```suggestion
                  Applications for the mentors of the ScholarX program 2023 will open on 13th of March 2023 and will close 11th of April 2023. 
                  Since the application period is over, you may submit your CV, a headshot, and expression of interest to <a
```